jerry-core/ecma/builtin-objects/ecma-builtin-json.c

```diff
@@ -730,7 +730,7 @@ ecma_builtin_json_parse_buffer (const lit_utf8_byte_t * str_start_p, /**< String
  * @return ecma value
  *         Returned value must be freed with ecma_free_value.
  */
-static ecma_value_t
+ecma_value_t
```

Comment:
What is the reason of this change?

Reply:
Nothing. It was a mistake. I didn't want to add this change.